pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs

```diff
@@ -149,10 +149,11 @@
           </div>
           <% } %>
         </div>
-  
+
         <table class="table table-sm table-hover">
           <thead>
             <tr>
+              <th width="50">Thumbnail</th>
```

Comment:
On student pages I think we shouldn't call out the thumbnail explicitly. That is, don't label the column "Thumbnail", but instead make the "Question" header span two columns to cover both the thumbnail and the question title.